packages/algolia/src/bin.ts

```diff
@@ -12,14 +12,16 @@ program
   .option('-o, --output [lockfilepath]', 'relative path to lock file', './algolia-lockfile.json')
   .option('-p, --publish')
   .requiredOption('-s, --source <source>')
-  .requiredOption('-d, --domain <domain>');
+  .requiredOption('-d, --domain <domain>')
+  .requiredOption('--sitemap-xml <sitemapXmlPath>');
```

Comment:
Do we need this option? It should be always under root/public/sitemap.xml?

Reply:
We can provide a default, but I'd keep it here.